src/graphql/queries/ProfileAddress.graphql

```diff
@@ -0,0 +1,5 @@
+query ProfileAddress($request: SingleProfileQueryRequest!) {
+  profile(request: $request) {
+    ownedBy
```

Comment:
I think you'll want `id` here as well. GraphQL response caching can get weird if you don't include it. Also, I will need it for my changes.

Reply:
Added! Great call on the Apollo cache.